DESIGN.md

```diff
@@ -0,0 +1,60 @@
+# Bulk Data Extract (BDE) Processor design
+
+## User requirements
+
+-  The _user interface_ is a read-only PostgreSQL database in AWS RDS, with
```

Comment:
We don't have Azure authentication for AWS available at LINZ unfortunately. But that would be nice.

Reply:
I don't understand. We're using `aws-azure-login` for the Geostore since forever.

Reply:
Right. So you mean using IAM with azure authentication. Unfortunately that means only people with access to an AWS account can access it (basically just developers). Ideally we would be able to connect any service to Active Directory directly without the need for an IAM account. So anyone at LINX could access it. But I guess can look at that if /when it is available.

Reply:
I wasn't aware that only developers had AWS access; I thought myapplications access to the Geostore was org wide. Anyway, we'll need to look into access control in more detail. Maybe even filter by IP, if nothing better is available.